### PR TITLE
CLI - Integrate system info for node

### PR DESCRIPTION
**Pull request by abuccts**
Integrate system info for node, add `sb node info` command.

### CODE REVIEW

File: superbench/cli/_commands.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def load_command_table(self, args):
             g.command('deploy', 'deploy_command_handler')
             g.command('exec', 'exec_command_handler')
             g.command('run', 'run_command_handler')
+        with CommandGroup(self, 'node', 'superbench.cli._node_handler#{}') as g:
```
Comment (cp5555) [inline superbench/cli/_commands.py:26]: What's the meaning of this?

File: superbench/cli/_commands.py [hunk 2]
```diff
@@ -23,6 +23,8 @@ def load_command_table(self, args):
...
+            g.command('info', 'info_command_handler')
```
Comment (cp5555) [inline superbench/cli/_commands.py:27]: How can we collect all information from all nodes?

File: superbench/cli/_commands.py (hunk 1 above)
Comment (abuccts) [reply superbench/cli/_commands.py:26]: create a command group https://github.com/microsoft/knack/blob/dev/docs/commands.md#commands

File: superbench/cli/_commands.py (hunk 2 above)
Comment (abuccts) [reply superbench/cli/_commands.py:27]: it can only be used for one node
the cli is even not installed on any compute node

Comment (cp5555) [reply superbench/cli/_commands.py:27]: Shall we add an ansible task for this? Maybe we can do it in future. I just think we can leverage this to check the configuration.

Comment (abuccts) [reply superbench/cli/_commands.py:27]: it's not aligned with current design (cli is only run in container, host does not install cli, but the collection needs to be run on host)
we can add such a feature later

**Review by cp5555 — APPROVED** [review]

**Review by yukirora — APPROVED** [review]

**Pull request merged**